docs/troubleshooting.md

```diff
@@ -0,0 +1,73 @@
+# Troubleshooting
+
+If you have problems, try to rebuild when compiling with `-DCMAKE_BUILD_TYPE=Debug`.
```

Comment:
-> `If you have problems, try rebuilding QLever in debug mode (`cmake . -DCMAKE_BUILD_TYPE=Debug -DLOGLEVEL=DEBUG && make -j4`)`